### DISCUSSION TITLE
Sort the presets?

### DISCUSSION THREAD

**Discussion by tweiner** [ZakKemble/AVRDUDESS#96] [Ideas]
Would it be possible to add the ability to sort the presets? 

Thanks
Tony W

**Comment by ZakKemble** [comment]
Aye, the presets stuff is clunky and needs sorting out. I'm not sure when I'll do it though. It's pretty easy to manually edit the presets.xml file though. It's in D:\Users\\(username)\AppData\Roaming\AVRDUDESS\
